### PR TITLE
Added wrfplot cli application

### PR DESCRIPTION
**Pull request by wxguy**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/wrfplot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - pip check
+    - wrfplot --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/wxguy/wrfplot
+  summary: Command line application to plot WRF model output data
+  dev_url: https://github.com/wxguy/wrfplot/
+  license: GPL-3.0 & GPL-1.0-or-later
```
Comment (dopplershift) [inline recipes/wrfplot/meta.yaml:49]:
```suggestion
  license: GPL-3.0 AND GPL-1.0-or-later
```

File: recipes/wrfplot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    - cartopy
+    - xarray
+    - matplotlib-base
+    - wrf-python >=1.3
+    - imageio
+    - tqdm
+    - netcdf4
+    - basemap
+    - wrapt
+    - numpy
```
Comment (dopplershift) [inline recipes/wrfplot/meta.yaml:36]: None of these are listed in the upstream package?

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just need a tweak to make the license metadata SPDX-compliant. Also had a question about the dependencies.

File: recipes/wrfplot/meta.yaml (hunk 2 above)
Comment (wxguy) [reply recipes/wrfplot/meta.yaml:36]: basemap was not listed in by upstream package. Without other two packages it would fail to build on windows.

**Pull request closed without merging**